### PR TITLE
add in courtfinder zone

### PR DESCRIPTION
**Pull request by cakeben**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/prod/courttribunalfinder-service-gov-uk.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ns: []
```
Comment (timja) [inline environments/prod/courttribunalfinder-service-gov-uk.yml:1]: you also need to add the zone in the prod directory, e.g. like:

https://github.com/hmcts/azure-public-dns/blob/master/components/prod/apply-divorce.tf

make sure to update all the resource names to your zone

Comment (cakeben) [reply environments/prod/courttribunalfinder-service-gov-uk.yml:1]: added in

**Review by timja — APPROVED** [review]
there's something wrong in the yaml file causing the build to fail but i can't see it by eyeballing should be easy to sort if you open in a text editor and compare it to another

File: environments/prod/courttribunalfinder-service-gov-uk.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+txt: 
+  - name: "_smtp._tls"
+    ttl: 3600
+    record: 
+      - "v=TLSRPTv1;rua=mailto:tls-rua@mailcheck.service.ncsc.gov.uk"
+
+cname:
+  - name: "hw2mzkztnolp"
+    ttl: 300
+    record:
```
Comment (timja) [inline environments/prod/courttribunalfinder-service-gov-uk.yml:20]: i don't think this is it, cnames can only have one target:
https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/apply-for-probate-service-gov-uk.yml#L17

**Review by timja — APPROVED** [review]

**Pull request merged**